lonabot/bot.py

```diff
@@ -340,15 +349,16 @@ def __init__(self, token: str, db: database.Database):
             return
 
         delta = None
+        zone = None
         m = re.match(r'(\d+):(\d+)', tz[1])
         if m:
             hour, mins = map(int, m.groups())
         else:
             try:
                 # TODO This won't consider daylight saving time BS
```

Comment:
We do now, though (at least slightly better). Perhaps we should remove the `TODO`(s)?

Reply:
It's partway there at least. Maybe an update of comments to reflect current status.